UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -187,6 +187,19 @@ public void CmdResist()
 		}
 	}
 
+	[Command]
+	public void CmdSlideItem(Vector3Int destination)
+	{
+		if (playerScript.IsInReach(destination, true) == false)
+		{
+			return;
+		}
+		PushPull pushPull = playerScript.pushPull.PulledObjectServer;
```

Comment:
Think you might need a null check here for this variable as PulledObjectServer could be null.